evm/src/cpu/kernel/asm/core/create_contract_account.asm

```diff
@@ -0,0 +1,51 @@
+// Create a smart contract account with the given address and the given endowment value.
+// Pre stack: value, address
+// Post stack: status
+%macro create_contract_account
```

Comment:
(This is pulled out of `process_txn.asm`, and generalized a bit so it can handle address collisions which can come up with `CREATE2`)